core/templates/services/assets-backend-api.service.ts

```diff
@@ -29,6 +29,8 @@ import { UrlInterpolationService } from 'domain/utilities/url-interpolation.serv
 import { Observable } from 'rxjs';
 import { CsrfTokenService } from 'services/csrf-token.service';
 
+import constants from 'assets/constants';
```

Comment:
We prefer using AppConstants over constants. All the constants present in the constants file are imported to the AppConstants file.

Reply:
Sorry for now being clear about my change... You can definitely keep those constants in the constants.ts file. But, when trying to access the constant value in the frontend code base, we use AppConstants over constants. constants is just a subset of AppConstants.